### PR TITLE
Retrieve network status - Closes #236

### LINKED ISSUE #236: Retrieve network status
### Description

Make sure network status can be retrieved with its properties.

Make the version 2 API able to retrieve data by various criteria.

The node constants have to be stored in memory.

Make sure that additional conditions are met according to the specification below.

### Motivation

- Access to those properties is required by the UI

### Acceptance Criteria

- Network status is able to retrieve by a specified API endpoint

#### Endpoints

- HTTP `/api/v2/network/status`
- RPC `get.network.status`

#### Request parameters

No params required.

#### Response example

200 OK

```jsonc
{
  "data": {
    "height": 27256,
    "finalizedHeight": 27112,
    "milestone": "0",
    "networkVersion": "2.0",
    "networkIdentifier": "08ec0e01794b57e5ceaf5203be8c1bda51bcdd39bb6fc516adbe93223f85d630",
    "reward": "500000000",
    "supply": "10094237000000000",
    "registeredModules": [“token”, “sequence”, “keys”, “dpos”, “legacyAccount”],
    "operations": [
      { "id": "2:0", "name": "token:transfer" }
      ...
    ],
    "blockTime": 10,
    "communityIdentifier": "Lisk",
    "maxPayloadLength": 15360,
  },
  "meta": {
    "lastUpdate": 123456789,
    "lastBlockHeight": 25,
    “lastBlockId”: 1354568
  },
  "links": {}
}

```

503 Service Unavailable
```jsonc
{
  "error": true,
  "message": "Service is not ready yet"
}

```

### PR DESCRIPTION
**Pull request by nagdahimanshu**
### What was the problem?
This PR resolves #236 

### How was it solved?

- [x] Register http-version2
- [x] HTTP /api/v2/network/status works
- [x] RPC get.network.status works
- [x] Response map properly
- [x] node constants stored in memory

### How was it tested?
Local

### CODE REVIEW

**Review by sameersubudhi — APPROVED** [review]
LGTM!

File: services/gateway/sources/version2/networkStatus.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            milestone: 'data.genesisConfig.rewards.milestones',
+            networkVersion: '=,string',
+            networkIdentifier: '=,string',
+            reward: '=,number',
+            supply: '=,string',
+            registeredModules: '=',
+            operations: '=',
+            blockTime: 'data.genesisConfig.blockTime,number',
+            communityIdentifier: 'data.genesisConfig.communityIdentifier,string',
+            minRemainingBalance: 'data.genesisConfig.minRemainingBalance,string',
```
Comment (reyraa) [inline services/gateway/sources/version2/networkStatus.js:33]: Thanks for adding.

File: services/gateway/sources/version2/networkStatus.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    params: {},
+    method: 'core.network.status',
+    definition: {
+        data: {
+            height: '=,string',
+            finalizedHeight: '=,number',
+            milestone: 'data.genesisConfig.rewards.milestones',
+            networkVersion: '=,string',
+            networkIdentifier: '=,string',
+            reward: '=,number',
```
Comment (MichalTuleja) [inline services/gateway/sources/version2/networkStatus.js:27]: As discussed:
1. `reward` -> `currentReward`
2. Pass `rewards` from Lisk Core.

Comment (nagdahimanshu) [reply services/gateway/sources/version2/networkStatus.js]: Done!

**Review by MichalTuleja — APPROVED** [review]

**Pull request merged**